helm/README.md

```diff
@@ -0,0 +1,45 @@
+### Helm Chart Instructions
+
+Helm allows us to manage Kubernetes applications. We define a 'Helm chart' and then 'Helm' manages creating, updating and destroying resources on a target Kubernetes cluster. To install Bailo using these guides you need the following prerequisites:
+
+- Helm (https://helm.sh/)
+- Kubectl (https://kubernetes.io/docs/tasks/tools/)
+- An existing AWS EKS cluster
+- Kubectl pointing to EKS cluster (check with `kubectl cluster-info`)
+
+All commands assume they are run in `helm/bailo` in the right context. Context can be set with:
```

Comment:
Minor, but perhaps slightly rephrase to "All commands assume they are run in the 'helm/bailo' directory" or something, just for clarification.